jupyter_ydoc/ydoc.py

```diff
@@ -193,6 +200,32 @@ def set(self, value):
             self._ymeta.set(t, "nbformat", nb["nbformat"])
             self._ymeta.set(t, "nbformat_minor", nb["nbformat_minor"])
 
+    def _get_empty_code_cell():
+        return {
+            "cell_type": "code",
+            "execution_count": None,
+            "metadata": {},
+            "outputs": [],
+            "source": "",
+            "id": str(uuid4()),
+        }
```

Comment:
Since it's not a method, it should be a free function outside of the class.
Ideally, we should use `nbformat` (see #46).